docs/DeveloperGuide.md

```diff
@@ -1060,6 +1058,31 @@ testers are expected to do more *exploratory* testing.
     (where incomplete details are given for the applicant being added).
     Expected: Error messages displaying the cause of error is shown in the result display.  
 
+### Editing an applicant
```

Comment:
Don't forget to add the link to this section in the table of contents!

Reply:
Done! 👍